examples/games/rock_paper_scissors/play_rock_paper_scissors.qnt

```diff
@@ -0,0 +1,57 @@
+// -*- mode: Bluespec; -*-
+module rock_paper_scissors {
+    // Constants for player names
+    const PLAYER1 : str
+    const PLAYER2 : str
+    // Enum types for the possible moves and game status
+    type Move = Init | Rock | Paper | scissors
+    type GameStatus = Started | Pending | Draw | Winner(str)
+    // Variables to hold the current state of each player's move and the game status
+    var p1State: Move
+    var p2State: Move
+    var status: GameStatus
+    def beats(m1: Move, m2: Move): bool = or {
+        (m1 == Rock and m2 == scissors),
+        (m1 == Paper and m2 == Rock),
+        (m1 == scissors and m2 == Paper),
+    }
+    // Initialize the game with both players in 'Init' state and game status as 'Started'
```

Comment:
If you use triple slashes `///` instead of double `//` on comments that immediately precede a definition, that comment is treated as a doc-comment. This means that, in your editor, if you hover over definition names, it will show this comment as the documentation for that name :smile:. Not super important, but cool to know.
```suggestion

    /// Initialize the game with both players in 'Init' state and game status as 'Started'
```